src/js/components/Button/StyledButton.js

```diff
@@ -162,6 +162,7 @@ const activeButtonStyle = (props) => css`
 
 const disabledButtonStyle = (props) => css`
   ${disabledStyle(props.theme.button.disabled.opacity)}
+  ${props.theme.global.input.disabled && props.theme.global.input.disabled}
```

Comment:
It seems odd to be applying an `input` styling here on button. I also think the theme object has changed. Can this be removed?